game.cpp

```diff
@@ -0,0 +1,111 @@
+#include "game.h"
+#include <cmath>
+
+Game::Game(int fieldSize){
+    gameField.createField(fieldSize);
+    winStatus = false;
+    loseStatus = false;
+}
+
+void Game::updateWinStatus(){
+    std::vector<std::vector<Cell>>& currentField = gameField.getField();
+    for(int i = 0; i < gameField.getSize(); i++){
+        for(int j = 0; j < gameField.getSize(); j++){
+            if(currentField[i][j].haveWinValue()){
```

Comment:
This  inner for loop could be replaced by STL algorithm to change value if condition is met